Mage.Sets/src/mage/cards/t/TheIndomitable.java

```diff
@@ -0,0 +1,112 @@
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.DealsDamageToAPlayerAllTriggeredAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.effects.AsThoughEffectImpl;
+import mage.abilities.effects.common.DrawCardSourceControllerEffect;
+import mage.abilities.keyword.CrewAbility;
+import mage.abilities.keyword.TrampleAbility;
+import mage.cards.Card;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.AsThoughEffectType;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.Outcome;
+import mage.constants.SetTargetPointer;
+import mage.constants.SubType;
+import mage.constants.SuperType;
+import mage.constants.Zone;
+import mage.filter.StaticFilters;
+import mage.filter.common.FilterControlledPermanent;
+import mage.filter.predicate.Predicates;
+import mage.filter.predicate.permanent.TappedPredicate;
+import mage.game.Game;
+
+/**
+ * @author jam736
+ */
+public final class TheIndomitable extends CardImpl {
+
+    public TheIndomitable(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId,setInfo,new CardType[]{CardType.ARTIFACT},"{2}{U}{U}");
+        this.supertype.add(SuperType.LEGENDARY);
+        this.subtype.add(SubType.VEHICLE);
+        this.power = new MageInt(6);
+        this.toughness = new MageInt(6);
+
+        // Trample
+        this.addAbility(TrampleAbility.getInstance());
+
+        // Crew 3
+        this.addAbility(new CrewAbility(3));
+
+        // Whenever a creature you control deals combat damage to a player, draw a card.
+        this.addAbility(new DealsDamageToAPlayerAllTriggeredAbility(
+                new DrawCardSourceControllerEffect(1),
+                StaticFilters.FILTER_CONTROLLED_A_CREATURE,
+                false, SetTargetPointer.NONE, true
+        ));
+
+        // You may cast The Indomitable from your graveyard as long as you control three or more tapped Pirates and/or Vehicles.
+        this.addAbility(new SimpleStaticAbility(Zone.GRAVEYARD, new TheIndomitableCastEffect()));
+    }
+
+    private TheIndomitable(final TheIndomitable card) {
+        super(card);
+    }
+
+    @Override
+    public TheIndomitable copy() {
+        return new TheIndomitable(this);
+    }
+}
+
+class TheIndomitableCastEffect extends AsThoughEffectImpl {
+    
+    private static final FilterControlledPermanent filter = new FilterControlledPermanent("three or more tapped pirates and/or vehicles");
+    static {
+        filter.add(TappedPredicate.TAPPED);
+        filter.add(Predicates.or(
+            SubType.PIRATE.getPredicate(),
+            SubType.VEHICLE.getPredicate()
+        ));
+    }
+
+    TheIndomitableCastEffect() {
+        super(AsThoughEffectType.PLAY_FROM_NOT_OWN_HAND_ZONE, Duration.EndOfGame, Outcome.Benefit);
+        staticText = "You may cast The Indomitable from your graveyard as long as you control three or more tapped Pirates and/or Vehicles.";
```

Comment:
don't hardcode card name, use `{this}` instead

Reply:
Thanks. Nice catch.